Styles/layout.css

```diff
@@ -12,5 +12,9 @@
 }
 
 #desktopNav{
+  display: flex;
+  justify-content: space-around;
+  min-height: 40px;
+  background-color: #6d7993;
   width: 100%;
 }
```

Comment:
I would recommend not capitalizing directory names. It can cause cross OS compatibility issues :)